openwisp_notifications/websockets/consumers.py

```diff
@@ -37,20 +44,48 @@ def disconnect(self, close_code):
     def send_updates(self, event):
         user = self.scope['user']
         # Send message only if notification belongs to current user
-        if event['recipient'] == str(user.pk):
-            unread_notifications = normalize_unread_count(
-                user.notifications.unread().count()
-            )
-            self.send(
-                json.dumps(
-                    {
-                        'type': 'notification',
-                        'notification_count': unread_notifications,
-                        'reload_widget': event['reload_widget'],
-                        'notification': event['notification'],
-                    }
+        if event['recipient'] != str(user.pk):
+            return
```

Comment:
This is the only line that has missing coverage and it is intentional. I have an idea in mind which will allow us to remove this condition from here. Hence, I didn't add a test for it. 
I will open a new issue for this with more details, once I test the idea.

Reply:
Have you tested the idea?

Reply:
https://github.com/openwisp/openwisp-notifications/issues/210#issue-1066359068

Reply:
Ok, let's work on that soonish.